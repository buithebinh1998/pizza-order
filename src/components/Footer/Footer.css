footer {
  width: 100%;
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: auto;
  background: linear-gradient(to right, #012333, #074969);
  color: white;
  font-weight: bold;
  text-align: center;
  overflow-x: hidden;
  flex-shrink: 0;
  margin-top: auto;
}

footer div {
  font-size: 1rem;
  margin: 0.3rem 0.2rem;
}

footer img {
  max-width: 100%;
  height: auto;
}

@media (max-width: 768px) {
  footer div {
    font-size: 0.8rem;
    margin: 0.2rem 0.1rem;
  }
  footer img {
    width: 80%;
  }
}

@media (max-width: 433px) {
  footer div {
    font-size: 0.45rem;
    margin: 0.1rem 0.05rem;
  }
  footer img {
    width: 60%;
  }
}
