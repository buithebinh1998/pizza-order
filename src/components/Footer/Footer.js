import React from 'react'
import './Footer.css'
import tel from '../../assets/img/tel.png'
import visa from '../../assets/img/visa.png'
const Footer = () => (
    <footer>
        <div>&copy; 2020 PYCOZZA VIETNAM COMPANY | PRIVACY POLICY</div>
        <div>  
        <div className="hotline">
            <div>HOTLINE DELIVERY</div>
            </div>
            <div className="img-container">
                <a href="tel:1900-2727"><img src={tel} alt=""/></a>
            </div>
        </div>
        
        <div className="img-container-corner">
            <img src={visa} alt=""/>
        </div>
    </footer>
)
export default Footer;