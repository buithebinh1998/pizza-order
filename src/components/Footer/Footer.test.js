import { testSnapshots } from "../../utils/test.util";

import Footer from "./Footer";

jest.mock('react-router-dom', () => ({
    NavLink: "NavLink"
}))

describe("<Footer / >", () => {
  testSnapshots(Footer, [
    {
      props: {},
      description: "default render"
    }
  ]);
});