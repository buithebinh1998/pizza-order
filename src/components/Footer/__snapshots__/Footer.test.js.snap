// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer / > snapshots default render 1`] = `
<footer>
  <div>
    Â© 2020 PYCOZZA VIETNAM COMPANY |
     
    <NavLink
      to="/policy"
    >
      PRIVACY POLICY
    </NavLink>
     
  </div>
  <div>
    <div
      className="hotline"
    >
      <div>
        HOTLINE DELIVERY
      </div>
    </div>
    <div
      className="img-container"
    >
      <a
        href="tel:1900-2707"
      >
        <img
          alt=""
          src="assets"
        />
      </a>
    </div>
  </div>
  <div
    className="img-container-corner"
  >
    <img
      alt=""
      src="assets"
    />
  </div>
</footer>
`;
